### ISSUE TITLE
Strange behaviour of AutoStatQ for posthole sensor

### ISSUE THREAD

**Issue by bosby74** [gesape/AutoStatsQ#40]
Hi Gesa,
 during some attempts to use AutostatQ to perform quality checks on a small temporary network in Northern Italy, I found strange behaviour in the software that I still have no answer to. 
The seismic network is composed of several equipment and as the first attempt I selected only the broadband stations. I can complete the step' sequence to the end for most of them but not for one station. This is a posthole station located at -252 m of depth and only for this one, I was not able to complete the restitution step before performing the checks. The config file is the same, stations.pf file is the same, inventory file is the same but only for this station I cannot get the data. I checked the XML file and all the information seems to be ok as well as the waveforms metadata, I also tried several combinations of pf file but the problems remain. Do you have an idea about the problem? Just to give you some more detail in the attachment you find the station.pf, inventory and configuration files that I'm using, the log file for the Restitution step I got and an example of an earthquake recorded by the station. Let me know if you have a suggestion about the problem...
Thank you in advance,
Simone
[div05.zip](https://github.com/user-attachments/files/16313495/div05.zip)

**Comment by gesape** [comment]
Hi Simone, there was a small typo that hindered the usage of HH1/HH2 components. Please pull an update and set all_channels: false in the config file. Then it should work also for this station,

**Comment by gesape** [comment]
It is not very clear from the instructions that 'all_channels' is only needed if a single station has multiple channel types, e.g. first EH* and later HH*. Will put it on the list of things to do...

**Comment by bosby74** [comment]
Dear Gesa,
 thank you a lot, the problem for DIV05 waveforms was solved using your trick (setting all_channel=false).
Running the code, however, I encountered another issue with the Gain calculation. In fact, this calculation gives only one event useful for the comparison between stations of my network and a close permanent station in the area (CH.MUGIO). I started from a deep event catalogue containing more than 100 events, so it is strange that only one passed the criterion for the gain calculation. In addition, also in this case, the results of DIV05 were excluded and no information about the gain for this station was provided. PSD and orientation tests gave more stable results but not the ones for the gain step. The same results also if I consider different methods such as synthetics or median comparison. Should I set up some other parameters to perform the analysis? Or is there some other information that the manual doesn't contain?
In attachment, you find the config file I used for

Thanks,
Simone

![gains_all_eventsZ_all_gains](https://github.com/user-attachments/assets/474cfdeb-cecb-4084-b0ed-d65e90a27a6e)
![gains_medianZ_map_log](https://github.com/user-attachments/assets/f3ff3e34-d526-46f0-b42d-96440c1cedbe)
[GAIN.log](https://github.com/user-attachments/files/16350847/GAIN.log)
[AutoStatsQ_settings-ALL.config.txt](https://github.com/user-attachments/files/16350854/AutoStatsQ_settings-ALL.config.txt)
